src/context/AppProvider/AppContext.tsx

```diff
@@ -185,8 +185,8 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
         dispatch(getValidatorData.initiate(accountId, options))
       }
 
-      await fetchAllOpportunitiesIds({ forceRefetch: true })
-      await fetchAllOpportunitiesMetadata({ forceRefetch: true })
+      await fetchAllOpportunitiesIds()
+      await fetchAllOpportunitiesMetadata()
```

Comment:
As per @0xdef1cafe : we should probably extract these 3 to its own function

We had also previously discussed rewriting this hook to a better accountIds/chainIds discrimination one, so that will probably be the best place to do it, as a follow-up